cmd/catchpointdump/database.go

```diff
@@ -70,6 +70,10 @@ var databaseCmd = &cobra.Command{
 		if err != nil {
 			reportErrorf("Unable to print key value store : %v", err)
 		}
+		err = printStateProofVerificationContext("./ledger.tracker.sqlite", outFile)
+		if err != nil {
+			reportWarnf("Unable to print state proof verification database : %v", err)
```

Comment:
We are using `reportWarnf ` to be backward compatible with the old ledger database.  `reportErrorf` exit the process, so if someone will run the new `catchpointdump` binary on an old ledger file, it will report an error, even though it is just old. Does it make sense?

Reply:
read CatchpointStateCatchupVersion form catchpointinfo and error if version exist and > 7